glossary.yml

```diff
@@ -127,6 +158,34 @@
       the term comes from a fairy tale in which children left a trail of breadcrumbs behind themselves
       so that they could find their way home.
 
+- slug: bug_report
+  en:
+    term: "bug report"
+    def: >
+      A collection of files, logs, or related information that adequately
+      describes either an unexpected output of some code or program or an unexpected
+      error or warning. This information is used to help find and fix a bug in the
+      program or code.
+
+- slug: bug_tracker
+  en:
+    term: "bug tracker"
+    def: >
+      A system that tracks and manages [reported bugs](#bug_report) for a software program,
```

Comment:
There are two methods for creating cross-links here. `[text](#ref)` and `[text](ref)` (https://github.com/carpentries/glosario/pull/15/files#diff-d85e4245bdeda1fb596b9ff8ade0690fR144). Which one is preferred? If these are on separate pages, I would assume that the `[text](ref)` syntax is preferred

Reply:
This is a bug in the example - all cross-references should use `[text](#ref)` so that they resolve in-page.

Reply:
Resolved with [text](#ref) as the standard.